app/src/main/java/grinnell/appdev/edu/lyles/fragments/Beers.java

```diff
@@ -0,0 +1,54 @@
+package grinnell.appdev.edu.lyles.fragments;
+
+/**
+ * Created by nannan on 2017/2/12.
+ */
+
+public class Beers {
+    private String title;
```

Comment:
These fields should follow the m + name convention, so for example `private String mTitle;`